x/ccv/consumer/keeper/grpc_query.go

```diff
@@ -11,9 +11,9 @@ import (
 	"github.com/cosmos/interchain-security/v3/x/ccv/consumer/types"
 )
 
-var _ types.QueryServer = Keeper{}
+var _ types.QueryServer = Keeper{} //nolint:golint
 
-func (k Keeper) QueryNextFeeDistribution(c context.Context,
+func (k Keeper) QueryNextFeeDistribution(c context.Context, //nolint:golint
```

Comment:
Why is exactly the linter complaining here? Is there a way to fix the linting issue without introducing `//nolint:glint`.

Reply:
the linter complains the keeper is not defined.